src/pcapi/emails/beneficiary_soon_to_be_expired_bookings.py

```diff
@@ -1,16 +1,38 @@
+from typing import List
+from typing import Tuple
+
+from pcapi.core.categories import subcategories
 from pcapi.core.users.models import User
 from pcapi.models import Booking
 
 
+def filter_books_bookings(bookings: list[Booking]) -> Tuple[List[Booking], List[Booking]]:
+
+    books_bookings = []
+    other_bookings = []
+
+    for booking in bookings:
+        stock = booking.stock
+        offer = stock.offer
+        if offer.subcategoryId == subcategories.LIVRE_PAPIER.id:
```

Comment:
```suggestion
        if offer.subcategoryId == subcategories.LIVRE_PAPIER.id and booking.dateCreated > BOOKS_BOOKINGS_AUTO_EXPIRY_DELAY_START_DATE:
```
Seules les réservations sur des livres créées après la date du début d'application du nouveau délai sont concernés par l'email de relance au bout de 5 jours

Reply:
est-ce que ce n'est pas redondant avec la fonction du repository qui va chercher les données de booking ?

Reply:
ok ce n'est pas redondant, j'ai inclus cet ajout

Reply:
Peut-on tester cette fonction `filter_books_bookings()` ?